### PR TITLE
Document configuration

### PR DESCRIPTION
**Pull request by Sohalt**

### CODE REVIEW

**Comment by suom1** [comment]
Replace `wireguard-ui` with `wg-ui` and it will be an good update! :)

**Comment by Sohalt** [comment]
Where should I replace `wireguard-ui` with `wg-ui`?
The environment variables need to be called `WIREGUARD_UI_<FLAG_NAME>`, because they are automatically generated by kingpin.

**Comment by suom1** [comment]
> The environment variables need to be called `WIREGUARD_UI_<FLAG_NAME>`, because they are automatically generated by kingpin.

I realized that it's only one line (the first line in documentation) you can change, as rest of it depends on our docker hub.

File: README.md [hunk 1]
```diff
@@ -24,6 +24,23 @@ The easiest way to run wireguard-ui is using the container image. To test it, ru
 
 When running in production, we recommend using the latest release as opposed to `latest`.
 
+### Configuration
+
+You can configure wireguard-ui using commandline flags or environment variables.
```
Comment (Sohalt) [inline README.md:29]: @suom1 so you want this changed to
```suggestion
You can configure wg-ui using commandline flags or environment variables.
```
?
If so, feel free to apply the change. But I don't really see the reason. The README also calls it "wireguard-ui" in the "Running" section.

**Pull request merged**